react-components/src/architecture/concrete/pointsOfInterest/color.ts

```diff
@@ -7,7 +7,7 @@ import { PointsOfInterestStatus } from './types';
 
 export const DEFAULT_OVERLAY_COLOR = new Color('#C945DB');
 export const PENDING_OVERLAY_COLOR = new Color('#33AA33');
-export const SELECTED_COLOR = new Color('##6464FF');
+export const SELECTED_COLOR = new Color('#5151ce');
```

Comment:
Woops - seems like I pushed the previous change a bit too fast :see_no_evil: Where did you find this new color? The previous one is the same we use for highlighting CAD, but perhaps it's a good idea to make them a little different :thinking:

Reply:
I used a color picker. Maybe the screenshot compression has changed the code